core/lib/Thelia/Controller/Admin/AdminLogsController.php

```diff
@@ -41,11 +43,12 @@ public function loadLoggerAjaxAction()
         ) as $entry) {
             $entries[] = array(
                 "head" => sprintf(
-                    "[%s][%s][%s:%s]",
+                    "[%s][%s][%s:%s%s]",
```

Comment:
why there is no white space before the new parameter ? Do you really want to keep it like this ?

Thanks